x-pack/plugins/security_solution/public/app/solution_navigation/links/sections/ml_links.ts

```diff
@@ -86,6 +86,7 @@ export const mlNavCategories: SolutionLinkCategory[] = [
       ExternalPageName.mlFileUpload,
       ExternalPageName.mlIndexDataVisualizer,
       ExternalPageName.mlDataDrift,
+      ExternalPageName.mlESQLdataVisualizer,
```

Comment:
Does this file control the order in which items appear in the left nav and on the ML page? The ES|QL data visualizer item should appear before the Data drift item, so I guess you just need to swap these two rows around?

Reply:
Updated in 209fbfc1a88a08ef1bbfde8d7e6e9a00a30705ee